components/service/glean/src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt

```diff
@@ -33,16 +43,37 @@ class GleanDebugActivity : Activity() {
 
         // Enable debugging options and start the application.
         intent.extras?.let {
+            // Check for ping debug view tag to apply to the X-Debug-ID header when uploading the
+            // ping to the endpoint
+            var pingTag: String? = intent.getStringExtra(TAG_DEBUG_VIEW_EXTRA_KEY)
+            var altUrl: String? = null
+
+            // Validate the ping tag against the regex pattern
+            pingTag?.let {
+                if (!pingTagPattern.matches(it)) {
+                    logger.error("tagPings value $it does not match accepted pattern $pingTagPattern")
+                    pingTag = null
+                } else {
+                    // Tagged pings are not to be sent to multiple endpoints, but must be redirected to
+                    // the endpoint that can handle them:
+                    // https://bugzilla.mozilla.org/show_bug.cgi?id=1522428#c7
+                    // https://bugzilla.mozilla.org/show_bug.cgi?id=1522428#c8
+                    altUrl = "http://stage.ingestion.nonprod.dataops.mozgcp.net"
```

Comment:
Should this be `https`?

Reply:
I copied verbatim what Arkadiusz posted in the Bugzilla comment

Reply:
Ah -- if that endpoint supports `https` (and it probably does) we should probably use that, though.

For the `MockWebServer`, (elsewhere in the PR) I think it's fine to use `http`.  It's (hopefully) safe to assume that our http client libraries have working `https` support, so I don't think we need to test that...

Reply:
I changed it to https and I'll verify that with Arkadiusz

Reply:
Yeah, this *MUST* be HTTPS. This must not land with http. Let's check with Arkadiusz, but the endpoint **must** support HTTPS :)

Reply:
Yes, please talk to us over https :)